airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/csv_parser.py

```diff
@@ -62,8 +62,16 @@ def read_data(
                 for row in reader:
                     # The row was not properly parsed if any of the values are None. This will most likely occur if there are more columns
                     # than headers or more headers dans columns
-                    if None in row or None in row.values():
-                        raise RecordParseError(FileBasedSourceError.ERROR_PARSING_RECORD)
+                    if None in row:
+                        raise RecordParseError(
```

Comment:
Should we add rows information so that the user can pinpoint where is the issue exactly in the file?

Reply:
Good call, added the line number.